frontend/src/api/requests.js

```diff
@@ -65,9 +66,12 @@ export const fetchBillsBlocks = async (
     pages.push(billIds.slice(i, i + PAGE_SIZE));
   }
 
+  const addedTerm = searchTerm !== '*' ? `AND ${searchTerm}` : '';
   for (let i = 0; i < pages.length; i++) {
     const page = pages[i];
-    const term = encodeURIComponent(`basePrintNo:(${page.join(' OR ')})`);
+    const term = encodeURIComponent(
+      `basePrintNo:(${page.map((printNo) => `+${printNo}`).join(' OR ')}) ${addedTerm}`
+    );
```

Comment:
```suggestion
    let term = `printNo:(${page.map((printNo) => `${printNo}`).join(' OR ')}) ${addedTerm}`;
    // term = encodeURIComponent(term);
    // console.log('term', term);
```

- I'm having issues w/ HTML encoding (request only works w/out)
- why the plus in `+${basePrintNo}` ?
- for some reason, I'm also getting versioned bill (this may be my fault, maybe we should use `printNo` instead of `basePrintNo` ..?)

Otherwise, it seems to work to me.

Reply:
the plus was an attempt to get it working with the legislator and status filters; i thought that applying some more specificity with the print num would let those filters actually filter by legislator or status. i dont think it's doing anything so ill remove

i'll also use `printNo` instead of `basePrintNo`, if it adds more specificity. im guessing "versioned bills" means non-canonical versions of the main bill?